CMakeLists.txt

```diff
@@ -24,8 +24,17 @@ if("${HALIDE_REQUIRE_LLVM_VERSION}" STREQUAL "")
   find_package(LLVM REQUIRED CONFIG)
 else()
   # Find a specific version.
-  string(SUBSTRING "${HALIDE_REQUIRE_LLVM_VERSION}" 0 1 MAJOR)
-  string(SUBSTRING "${HALIDE_REQUIRE_LLVM_VERSION}" 1 1 MINOR)
+  string(REGEX REPLACE
```

Comment:
Does llvm even use minor versions anymore?

Reply:
I don't know.

Reply:
Or, to put it another, is it enough for us to just check the major version?

Reply:
According to http://blog.llvm.org/2016/12/llvms-new-versioning-scheme.html: no, it will always be zero henceforth, so the version will be MAJOR.zero.patch. Want me to revise?

Reply:
I mean, it's breaking the build right? So only if you have time to kill.